README.md

```diff
@@ -10,36 +10,46 @@
 - `npm run test` - run JS tests
 - `npm run watch` - start a file watcher that runs tests & lint
 - `npm run prettier` - clean up JS formatting
-- `npm run deploy:dev -- --stage lmorchard` - deploy a stack with stage `lmorchard` configured for development (e.g. with `ENABLE_DEV_AUTH=1`)
-- `npm run logs -- --stage lmorchard -f accept -t` - watch logs for the function `accept`
-- `npm run deploy -- --stage production` - deploy a stack with stage `production` configured for production
+- `npm run deploy` - deploy a stack configured for production
+- `npm run deploy:dev` - deploy a stack configured for development (e.g. with `ENABLE_DEV_AUTH=1`)
+- `npm run info` - display information about the currently deployed stack (e.g. handy for checking the stack's API URL)
+- `npm run logs -- -f accept -t` - watch logs for the function `accept`
 - `npm run client -- [--id <id> --key <key>] <url>` - make an authenticated request to `<url>` using Hawk credentials, defaults to dev credentials devuser / devkey enabled with `ENABLE_DEV_AUTH` env var on deploy
 
-### Quickstart Notes for Stackless on AWS
+### Quickstart Notes
 
-Development is currently done directly on Amazon Web Services. So, you'll need to [sign up for an account](https://aws.amazon.com/) or [request a Dev IAM account from Mozilla Cloud Operations](https://mana.mozilla.org/wiki/display/SVCOPS/Requesting+A+Dev+IAM+account+from+Cloud+Operations). (The latter is available only to Mozillians.)
+First, ensure [node.js 8.11.1](https://nodejs.org/en/) or newer is installed. Then, the steps to get started look something like this:
```

Comment:
AWS Lambda is running node.js 8.10, so maybe we shouldn't run something newer than 8.x locally.